packages/random/_test.pony

```diff
@@ -239,3 +243,430 @@ class iso _TestRandomShuffle is UnitTest
     h.assert_eq[String](words(6)?, "jumps")
     h.assert_eq[String](words(7)?, "brown")
     h.assert_eq[String](words(8)?, "lazy")
+
+class iso _TestXorOshiro128StarStar is UnitTest
+  fun name(): String => "random/xoroshiro128**"
+
+  fun apply(h: TestHelper) =>
+    let xoroshiro128 = XorOshiro128StarStar(5489)
+    h.assert_eq[U64](xoroshiro128.next(), 529225608228480)
```

Comment:
Would it be possible (and would you consider it worth doing) to instead iterate over a list of U64 instead of repeating this line over and over?

Reply:
Also, are these test vectors from the papers, or from some other implementation? I suppose adding a comment where the numbers come from could be useful in the future.

Reply:
The numbers are from executions of the reference implementation in C. I added comments about each source.

I also initially wanted to use a list to iterate over, but i decided against it, as the effort would have been roughly the same and the current version has the advantage that the error message in case of failure will tell you at which position it failed, without counting lines while iterating through a list.

Reply:
Thanks! You're right about that line no detail, what you've chosen is better! =)